functions/src/Controller/Elections.ts

```diff
@@ -1,3 +1,3 @@
 import { Elections, electionsConverter } from '../Models/Elections';
 import * as functions from 'firebase-functions';
-import { firestore } from 'firebase-admin';
+import { db } from 'firebase-admin';
```

Comment:
db isn't imported from firebase-admin, it's imported from the index file, db is our own instantiated firestore that we declare inside index.ts, so replace this with 
"import { db } from '../index';"

Reply:
done